src/InertiaJsPreset.php

```diff
@@ -23,16 +23,14 @@ protected static function updatePackageArray(array $packages)
     {
         return array_merge([
             '@babel/plugin-syntax-dynamic-import' => '^7.2.0',
-            'inertia' => 'github:inertiajs/inertia',
-            'inertia-vue' => 'inertiajs/inertia-vue',
+            '@inertiajs/inertia' => '^0.1',
+            '@inertiajs/inertia-vue' => '^0.1',
```

Comment:
Not sure it's a big deal, but I've noticed that NPM automatically added the third number to the version: `^0.1.0`

Reply:
Probably not, but i've updated anyway. I think that when we give you a preset, running commands (`npm`) shouldn't change what we provide you.

Reply:
Yeah man agreed, but 🤷‍♂️

MURGEIT